### PR TITLE
Adding binary builds to circleci

### PR DESCRIPTION
**Pull request by pjh5**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1478,123 +2822,309 @@ jobs:
...
   version: 2
   build:
     jobs:
-      - pytorch_linux_trusty_py2_7_9_build
-      - pytorch_linux_trusty_py2_7_9_test:
-          requires:
-            - pytorch_linux_trusty_py2_7_9_build
-      - pytorch_linux_trusty_py2_7_build
-      - pytorch_linux_trusty_py2_7_test:
+      - binary_linux_manywheel_2.7mu_cpu_build
```
Comment (yf225) [inline .circleci/config.yml]: To access the `org-member` variables, we should likely put `context: org-member` under this item (at the same indentation level as `requires:`.

Comment (pjh5) [reply .circleci/config.yml]: This worked. I wonder why I can't specify this at the job definition level

File: tools/build_pytorch_libs.sh [hunk 2]
```diff
@@ -298,16 +298,16 @@ function build_caffe2() {
...
-      # root/torch/lib/tmp_install/lib
-      report "Updating all install_names in $INSTALL_DIR/lib"
-      pushd "$INSTALL_DIR/lib"
-      for lib in *.dylib; do
-          report "Updating install_name for $(pwd)/$lib"
-          install_name_tool -id @rpath/$lib $lib
-      done
-      popd
-  fi
+  #if [[ $(uname) == 'Darwin' ]]; then
```
Comment (pjh5) [inline tools/build_pytorch_libs.sh]: This was going to be removed by Anders in an earlier PR but I asked him to wait until I tested it here with the binaries. If all the Mac jobs pass then I will remove this

**Review by yf225 — APPROVED** [review]
Looks good to me, although it would be ideal to reduce the number of lines in the `config.yml` file :)

**Pull request closed without merging**